### PR TITLE
Add note about enabling container registry

### PR DESCRIPTION
**Pull request by charlieegan3**

### CODE REVIEW

File: src/writing-policies/rust/02-create-policy.md [hunk 1]
```diff
@@ -69,3 +69,7 @@ The command will produce the following output:
 ```
 
 The new policy project can now be found inside of the `demo` directory.
+
+Note: if you plan to make sure of the GitHub container registry
```
Comment (ereslibre) [inline src/writing-policies/rust/02-create-policy.md:73]:
```suggestion
Note: if you plan to make use of the GitHub container registry
```

**Review by ereslibre — CHANGES_REQUESTED** [review]
Thanks! Minor typo.

**Comment by charlieegan3** [comment]
No probs. I've updated that now.

**Review by ereslibre — APPROVED** [review]
Thanks a lot!

**Pull request merged**